src/SvelteTable.svelte

```diff
@@ -75,6 +92,36 @@
   const handleClickCell = (event, row, key) => {
     dispatch('clickCell', { event, row, key });
   };
+
+  const handlePageClick = (direction) => {
```

Comment:
to follow the pattern this should be called handleClickPage, also the event should be changePage